diff --git a/include/target.mk b/include/target.mk
index 530e711fa5..d58f08c0e5 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -168,7 +168,7 @@ LINUX_RECONF_DIFF = $(SCRIPT_DIR)/kconfig.pl - '>' $(call __linux_confcmd,$(filt
 ifeq ($(DUMP),1)
   BuildTarget=$(BuildTargets/DumpCurrent)
 
-  CPU_CFLAGS = -Os -pipe
+  CPU_CFLAGS = -O3 -pipe
   ifneq ($(findstring mips,$(ARCH)),)
     ifneq ($(findstring mips64,$(ARCH)),)
       CPU_TYPE ?= mips64
@@ -209,7 +209,7 @@ ifeq ($(DUMP),1)
   endif
   ifeq ($(ARCH),aarch64)
     CPU_TYPE ?= generic
-    CPU_CFLAGS_generic = -mcpu=generic
+    CPU_CFLAGS_generic = -march=armv8-a+crypto+crc -mcpu=cortex-a73.cortex-a53+crypto+crc -mtune=cortex-a73.cortex-a53
     CPU_CFLAGS_cortex-a53 = -mcpu=cortex-a53
   endif
   ifeq ($(ARCH),arc)
diff --git a/package/ctcgfw/luci-app-turboacc/Makefile b/package/ctcgfw/luci-app-turboacc/Makefile
index 7e07b87fee..a08cc4dbe3 100644
--- a/package/ctcgfw/luci-app-turboacc/Makefile
+++ b/package/ctcgfw/luci-app-turboacc/Makefile
@@ -22,7 +22,7 @@ LUCI_PKGARCH:=all
 define Package/$(PKG_NAME)/config
 config PACKAGE_$(PKG_NAME)_INCLUDE_flow-offload
 	bool "Include Flow Offload"
-	depends on PACKAGE_$(PKG_NAME)_INCLUDE_shortcut-fe=n
+	#depends on PACKAGE_$(PKG_NAME)_INCLUDE_shortcut-fe=n
 	default y
 
 config PACKAGE_$(PKG_NAME)_INCLUDE_shortcut-fe
diff --git a/package/ctcgfw/luci-app-turboacc/root/etc/config/turboacc b/package/ctcgfw/luci-app-turboacc/root/etc/config/turboacc
index 067aad2763..b221186ba4 100644
--- a/package/ctcgfw/luci-app-turboacc/root/etc/config/turboacc
+++ b/package/ctcgfw/luci-app-turboacc/root/etc/config/turboacc
@@ -1,9 +1,9 @@
 
 config turboacc 'config'
 	option sw_flow '1'
-	option hw_flow '1'
-	option sfe_flow '1'
-	option sfe_bridge '1'
+	option hw_flow '0'
+	option sfe_flow '0'
+	option sfe_bridge '0'
 	option sfe_ipv6 '0'
 	option bbr_cca '1'
 	option fullcone_nat '1'
diff --git a/package/ctcgfw/luci-app-vssr/root/usr/share/vssr/genconfig_trojan.lua b/package/ctcgfw/luci-app-vssr/root/usr/share/vssr/genconfig_trojan.lua
index 875813dc08..f2e186e9c2 100644
--- a/package/ctcgfw/luci-app-vssr/root/usr/share/vssr/genconfig_trojan.lua
+++ b/package/ctcgfw/luci-app-vssr/root/usr/share/vssr/genconfig_trojan.lua
@@ -21,8 +21,8 @@ local trojan = {
         verify = (server.insecure == '0') and true or false,
         verify_hostname = (server.tls == '1') and true or false,
         cert = '',
-        cipher = 'ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA:AES128-SHA:AES256-SHA:DES-CBC3-SHA',
-        cipher_tls13 = 'TLS_AES_128_GCM_SHA256:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_256_GCM_SHA384',
+        cipher = 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA:AES128-SHA:AES256-SHA:DES-CBC3-SHA',
+        cipher_tls13 = 'TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:TLS_AES_256_GCM_SHA384',
         sni = server.peer,
         alpn = {'h2', 'http/1.1'},
         curve = '',
diff --git a/package/lean/luci-app-cpufreq/root/etc/config/cpufreq b/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
index 5c2c070e9d..e1ce69b52e 100644
--- a/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
+++ b/package/lean/luci-app-cpufreq/root/etc/config/cpufreq
@@ -1,3 +1,9 @@
 
 config settings 'cpufreq'
+	option governor0 'schedutil'
+	option maxfreq0 '1608000'
+	option governor4 'schedutil'
+	option maxfreq4 '2016000'
+	option minfreq0 '600000'
+	option minfreq4 '600000'
 